dataloadgen.go

```diff
@@ -127,11 +148,11 @@ func (l *Loader[KeyT, ValueT]) LoadThunk(key KeyT) func() (ValueT, error) {
 
 // LoadAll fetches many keys at once. It will be broken into appropriate sized
 // sub batches depending on how the loader is configured
-func (l *Loader[KeyT, ValueT]) LoadAll(keys []KeyT) ([]ValueT, []error) {
+func (l *Loader[KeyT, ValueT]) LoadAll(ctx context.Context, keys []KeyT) ([]ValueT, []error) {
```

Comment:
You added "l.tracer.Start(ctx, "LOAD")" to load, but not here? Why? Should we even bother with spans around the higher level methods or maybe we care only about instrumenting LoadThunk?

Reply:
sorry, inadvertently i was focusing on `Load` for this draft just to get everything working, when it's more stable i'll apply it to `LoadAll` as well. but i do agree that `LoadThunk` probably makes more sense.